### PR TITLE
Title + Stevedore rebranding + Keys changes

### PR DESCRIPTION
**Pull request by BadAtThisGame302**
## About The Pull Request

Removes the Merchant key from the Stewardry and replaces it with a Nitemaiden key.

Removes the Merchant key from the Hand and replaces it with the Captain's key.

Renames the Shepard keyring to Adept keyring. (Given it's given to the Adepts)

~~Moves Niteman to the noble positions~~ and adds a Concubine/Champion of Rockhill/Master of the Forge title.

Gives the Stevedore's a do-over, descriptions and map changed to accomodate this.

Adds 2 other unused sprites to the chance of the gold statuette.

Added an unused variant for the statue of Astrata that features more jewerly in the code. No reason for it to be left unused.

## Why It's Good For The Game

Most of the community is on the side of the Crown not getting Merchant keys so this finalizes it.

Renames to keyring is QOL ~~aswell as moving the *noble* Niteman to the noble positions.~~ Also the Concubine is a title along with Consort I originally wanted to add but withheld Consort for now, kept Concubine. Also added Champion of Rockhill for any true PATRIOT that did a heroic deed and deserves to be venerated. Likewise the Master of the Forge is a smith that's PEAKED at smithing and deserves to be YOUR boss.

Merchant is generally nowadays after it was removed from the required roles, not played much or killed because he's charging too much so that others may use his GOLDFACE freely. Pushing the Mercenaries to him didn't work either because of no mercs or no merchant or mercenaries working for the Crown more than him. So made him the Merchant's goons and pray that this works out in the end.

## Pre-Merge Checklist

- [ ] You tested this on a local server.
- [X] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by NPC1314** [comment]
Makes sense

**Comment by NPC1314** [comment]
Actually if you gonna noble niteman you gotta remove drow and tief

**Comment by BadAtThisGame302** [comment]
> Actually if you gonna noble niteman you gotta remove drow and tief

Feldsher and Merchant both have the noble trait and are in noble_positions and are open to all jobs.

**Comment by ThePainkiller** [comment]
> > Actually if you gonna noble niteman you gotta remove drow and tief
> 
> Feldsher and Merchant both have the noble trait and are in noble_positions and are open to all jobs.

Oversights, not standards

**Comment by BadAtThisGame302** [comment]
> Actually if you gonna noble niteman you gotta remove drow and tief

Removed Dark Elf and Tiefling from Niteman. Also updated the description of the role.

**Comment by Mogelix** [comment]
> > Actually if you gonna noble niteman you gotta remove drow and tief
> 
> Removed Dark Elf and Tiefling from Niteman. Also updated the description of the role.

This will be unpopular, resented, and negative for the bathhouse as a whole. What already few people play nitematron, I notice many enjoy playing a shunned noble also as a discriminated race. Deeming this an 'oversight' and favouring 'standardization' seemingly for its own sake is a hallmark of inflexibility: it is exceptions that prove rules!

**Comment by BadAtThisGame302** [comment]
> > > Actually if you gonna noble niteman you gotta remove drow and tief
> > 
> > 
> > Removed Dark Elf and Tiefling from Niteman. Also updated the description of the role.
> 
> This will be unpopular, resented, and negative for the bathhouse as a whole. What already few people play nitematron, I notice many enjoy playing the already shunned noble as a discriminated race. Deeming this an 'oversight' and justifying with 'standardization' seemingly for its own sake is a hallmark of inflexibility: it is exceptions that prove rules!

I 100% agree, already said so in the discord but I'll say it again. Niteman doen't deserve the normal standardization of race blocking. It's mostly played by those two races ironically enough.

**Comment by NPC1314** [comment]
Why not add tief and drow kings then? Why do they deserve to be blocked?
If theyre nobles they arent shunned, they are in fact the social elite, this is basic common sense.

If you want to RP being a disgraced noble do it, but youre enforcing them being nobles in code, why?
Whats the point? If them being drow or tieflings why not keep that and skip ennobling them?

**Comment by BadAtThisGame302** [comment]
> Why not add tief and drow kings then? Why do they deserve to be blocked? If theyre nobles they arent shunned, they are in fact the social elite, this is basic common sense.
> 
> If you want to RP being a disgraced noble do it, but youre enforcing them being nobles in code, why? Whats the point? If them being drow or tieflings why not keep that and skip ennobling them?

Niteman was already a noble, he had the trait. I'm just moving them to noble positions. Tief and Drow is removed but I don't agree with it, most Nitematrons are played by those two races anyways.

**Comment by NPC1314** [comment]
> > Why not add tief and drow kings then? Why do they deserve to be blocked? If theyre nobles they arent shunned, they are in fact the social elite, this is basic common sense.
> > If you want to RP being a disgraced noble do it, but youre enforcing them being nobles in code, why? Whats the point? If them being drow or tieflings why not keep that and skip ennobling them?
> 
> Niteman was already a noble, he had the trait. I'm just moving them to noble positions. Tief and Drow is removed but I don't agree with it, most Nitematrons are played by those two races anyways.

Just saying Niteman is already a noble isnt a good argument, people add traits and change stuff all the time on whims, you should be able to motivate things on something better than "its how it is now".

If you think Drow and Tiefling are so important why do you insist on moving them to nobiliy, why not remove noble trait instead?
Doing it this way is strange.

**Comment by BadAtThisGame302** [comment]
@ThePainkiller @NPC1314 I removed the Niteman change from noble_positions and also removed the description to say they're a courtier, brought back Dark-Elves and Tieflings and kept noble trait.

TLDR? Status Quo.

**Review by NPC1314 — APPROVED** [review]
